src/search/index.jsx

```diff
@@ -14,33 +14,36 @@ export class Search extends Component {
   }
 
   render() {
+    const name = this.props.name;
+
     return (
-      <ApplyChanges
-        type="form"
-        onApply={() => this.emitChange()}
-      >
-        <div className="govuk-grid-row">
-          <div className="govuk-grid-column-two-thirds">
-            <div className="govuk-form-group search-box">
-              <label className="govuk-label" htmlFor="filter">{this.props.label || 'Search'}</label>
-              { this.props.hint && <span className="govuk-hint">{this.props.hint}</span> }
-              <input
-                className="govuk-input"
-                id="filter"
-                name="filter"
-                type="text"
-                value={ this.state ? this.state.value : this.props.filter }
-                onChange={e => this.setState({ value: e.target.value })}
-              />
-              <button type="submit" className="govuk-button"></button>
-            </div>
-          </div>
+      <ApplyChanges type="form" onApply={() => this.emitChange()}>
+        <div className="govuk-form-group search-box">
+          { !this.props.hideLabel &&
+            <label className="govuk-label" htmlFor={name}>{this.props.label}</label>
+          }
```

Comment:
This might be nicer if it was just checking if the label prop exists and only shows it if it does.

i.e. if you don't want a label, set `label={false}` rather than having an extra property.

Reply:
done.